extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -190,6 +199,46 @@ class JetpackContactForm extends Component {
 		);
 	}
 
+	renderConfirmationMessageFields() {
+		const { instanceId } = this.props;
+		const { customThankyou, customThankyouMessage, customThankyouRedirect } = this.props.attributes;
+		return (
+			<Fragment>
+				<SelectControl
+					label={ __( 'On Submission', 'jetpack' ) }
```

Comment:
Was the use of title case intentional here? In Jetpack we typically use sentence case; p6TEKc-2Lx-p2

Reply:
WordPress generally uses title case in the sidebar, but I have no strong preference about whether we use title or sentence case here. @lessbloat?

Reply:
Yeah, I was only following the conventions used in the color section, which I assume came from core: 

![Title-Case](https://user-images.githubusercontent.com/5634774/66958501-4818dd00-f036-11e9-80d1-d7f6575fbe69.png)

Reply:
I'll leave it as is for this PR, as it fits in with the surrounding panels. There's probably a broader discussion to be had about ensuring everything fits into the place they're in.